packages/patternfly-3/patternfly-react/src/components/CopyToClipboard/CopyToClipboardBlockBar.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Panel } from 'react-bootstrap';
+import { Form, Icon, Button, OverlayTrigger, Tooltip } from '../..';
+
+export const CopyToClipboardBlockBar = props => {
+  const { value, onChange, onCopy, onClick, copied, show } = props;
+  const iconName = copied ? 'check' : 'paste';
+  const tooltip = (
+    <Tooltip id={copied ? 'tooltip-copied' : 'tooltip-copy-to-clipboard'}>
+      {copied ? 'Copied' : 'Copy to Clipboard'}
+    </Tooltip>
+  );
+  return (
+    <Form.FormGroup>
+      <Form.InputGroup>
+        <Form.InputGroup.Button>
+          <Button onClick={onClick}>
+            <Icon type="fa" name={show ? 'angle-down' : 'angle-right'} />
+          </Button>
+        </Form.InputGroup.Button>
+        <Form.FormControl
+          bsClass="form-control text-overflow-pf"
+          type="text"
+          value={value}
+          onChange={onChange}
+          onCopy={onCopy}
+        />
+        <Form.InputGroup.Button>
+          <OverlayTrigger placement="top" overlay={tooltip}>
+            <Button onClick={onCopy}>
```

Comment:
Using a screen reader, this button read aloud "Copy to Clipboard button" when focused, which I think comes from the Tooltip text. This is on the right track (more descriptive than just "button") but still has an issue. If there were multiple copy controls on a same page, how would a visually impaired user differentiate between them?

Might be good for the example to showcase setting a proper accessible name that describes what exactly is going to be copied. See the "A11y" tab on https://www.patternfly.org/patternfly-ng/#/inlinecopy for a full example that showcases the problem and an elegant solution.

<img width="1207" alt="screen shot 2018-11-27 at 10 35 28 am" src="https://user-images.githubusercontent.com/5942899/49094972-7f6f7700-f235-11e8-932c-b530e2a0c72c.png">